Update availability-zones-overview.md (#125048)
Azure/azure-content: articles/reliability/availability-zones-overview.md

## Physical and logical availability zones

> Each datacenter is assigned to a physical zone. Physical zones are mapped to logical zones in your Azure subscription, and different subscriptions might have a different mapping order. Azure subscriptions are automatically assigned their mapping at the time the subscription is created. Because of this, the zone mapping for one subscription could be different for other subscriptions. For example: Subscription A may have physical zone X mapped to logical zone 1, while subscription B has physical zone X mapped to logical zone 3, instead. You can use the dedicated ARM API called: [checkZonePeers](/rest/api/resources/subscriptions/check-zone-peers) to compare zone mapping for resilient solutions that span across multiple subscriptions. To be able to use this API the feature "Microsoft.Resources/AvailabilityZonePeering" needs to be enabled. For more information on how to enable features you can follow the steps in [Register features in Azure subscription](https://learn.microsoft.com/azure/azure-resource-manager/management/preview-features).

Reviewer: ```suggestion
Each datacenter is assigned to a physical zone. Physical zones are mapped to logical zones in your Azure subscription, and different subscriptions might have a different mapping order. Azure subscriptions are automatically assigned their mapping at the time the subscription is created. Because of this, the zone mapping for one subscription could be different for other subscriptions. For example: Subscription A may have physical zone X mapped to logical zone 1, while subscription B has physical zone X mapped to logical zone 3, instead. You can use the dedicated ARM API called: [checkZonePeers](/rest/api/resources/subscriptions/check-zone-peers) to compare zone mapping for resilient solutions that span across multiple subscriptions. To be able to use this API the feature "Microsoft.Resources/AvailabilityZonePeering" needs to be enabled. For more information on how to enable features you can follow the steps in [Register features in Azure subscription](/azure/azure-resource-manager/management/preview-features).
```

> Each datacenter is assigned to a physical zone. Physical zones are mapped to logical zones in your Azure subscription, and different subscriptions might have a different mapping order. Azure subscriptions are automatically assigned their mapping at the time the subscription is created. Because of this, the zone mapping for one subscription could be different for other subscriptions. For example: Subscription A may have physical zone X mapped to logical zone 1, while subscription B has physical zone X mapped to logical zone 3, instead. You can use the dedicated ARM API called: [checkZonePeers](/rest/api/resources/subscriptions/check-zone-peers) to compare zone mapping for resilient solutions that span across multiple subscriptions. To be able to use this API the feature "Microsoft.Resources/AvailabilityZonePeering" needs to be enabled. For more information on how to enable features you can follow the steps in [Register features in Azure subscription](https://learn.microsoft.com/azure/azure-resource-manager/management/preview-features).

Reviewer 2: ```suggestion
Each datacenter is assigned to a physical zone. Physical zones are mapped to logical zones in your Azure subscription, and different subscriptions might have a different mapping order. Azure subscriptions are automatically assigned their mapping at the time the subscription is created. Because of this, the zone mapping for one subscription could be different for other subscriptions. For example: Subscription A may have physical zone X mapped to logical zone 1, while subscription B has physical zone X mapped to logical zone 3, instead. 

To compare zone mapping for resilient solutions that span across multiple subscriptions, use the dedicated ARM API [checkZonePeers](/rest/api/resources/subscriptions/check-zone-peers). To use the `checkZonePeers` API, the feature "Microsoft.Resources/AvailabilityZonePeering" needs to be enabled. For more information on how to enable features, see [Register features in Azure subscription](https://learn.microsoft.com/azure/azure-resource-manager/management/preview-features).
```
